src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Commands/Create/CreateDialogCommand.cs

```diff
@@ -44,7 +44,7 @@ public async Task<CreateDialogResult> Handle(CreateDialogCommand request, Cancel
     {
         if (!await _userService.CurrentUserIsOwner(request.ServiceResource, cancellationToken))
         {
-            return new Unauthorized();
+            return new Forbidden(new[] { $"??????Not owner of {request.ServiceResource}??????" });
```

Comment:
@elsand 
Skal det være en egen problem detail-oppføring her for "ikke eier av ServiceResource"?

Reply:
Jepp, foreløpig kan teksten være "Not owner of {request.ServiceResource}". Her blir det naturlig å legge inn mer informasjon ifm https://github.com/digdir/dialogporten/issues/40